Bottom: 465695c41ceed4d41aaebf0e135b8f1123912988
Top:    ec44285035bdee25fdfdd25c2385cb08deeced27
Author: Alexandre Oliva <aoliva@redhat.com>
Date:   2013-08-16 00:24:28 -0300

Refresh of manual-document-mt-safety.patch

---

diff --git a/manual/intro.texi b/manual/intro.texi
index 07f27c9..d183e12 100644
--- a/manual/intro.texi
+++ b/manual/intro.texi
@@ -308,9 +308,10 @@ locking by the caller are the entire family of stdio functions.
 @cindex xguargs
 
 Functions marked with @code{xguargs} may use or modify objects passed as
-arguments without any guards to ensure consistency.  To ensure MT- and
-AS-Safe behavior, callers must ensure that the objects passed in are not
-modified concurrently by other threads or signal handlers.
+arguments without any guards that could guarantee consistency.  To
+ensure MT- and AS-Safe behavior, callers must ensure that the objects
+passed in are not modified concurrently by other threads or signal
+handlers.
 
 
 @item @code{tempchwd}
@@ -572,7 +573,7 @@ require a function to be Safe, but our implementation is Safe, or
 vice-versa.
 
 For the time being, the absence of this remark does not imply the safety
-propertes we documentated are identical to those mandated by POSIX for
+properties we documented are identical to those mandated by POSIX for
 the corresponding functions.
 
 @end itemize
